docs/_data/table/en/user-configuration.yml

```diff
@@ -30,6 +30,9 @@ alluxio.user.file.worker.client.threads:
   How many threads to use for file worker clients to read from workers.
 alluxio.user.file.write.location.policy.class:
   The default location policy for choosing workers for writing a file's blocks
+alluxio.user.file.write.capacity.reserved.ratio:
```

Comment:
The property is discussed in this pull request(https://github.com/Alluxio/alluxio/pull/4423), Because my git commit history  is corrupted, So, I open the  new pull request.